packages/SwingSet/test/vat-admin/terminate/terminate.test.js

```diff
@@ -475,18 +475,16 @@ test.serial('dead vat state removed', async t => {
   // have refCount=2 (one for bootstrap, one for the dynamic vat)
   let decidedKPID;
   let subscribedKPID;
-  for (const key of enumeratePrefixedKeys(kvStore, 'v6.')) {
-    if (key.startsWith('v6.c.kp')) {
-      const kpid = key.slice('v6.c.'.length);
-      const refCount = Number(kvStore.get(`${kpid}.refCount`));
-      const decider = kvStore.get(`${kpid}.decider`);
-      if (decider === 'v6') {
-        decidedKPID = kpid;
-      } else {
-        subscribedKPID = kpid;
-      }
-      t.is(refCount, 2, `${kpid}.refCount=${refCount}, not 2`);
+  for (const { suffix } of enumeratePrefixedKeys(kvStore, 'v6.c.kp')) {
+    const kpid = `kp${suffix}`;
```

Comment:
I think we should preserve the existing behavior, at least in this PR.
```suggestion
  for (const { suffix } of enumeratePrefixedKeys(kvStore, 'v6.')) {
    if (!suffix.startsWith('c.kp')) continue;
    const kpid = stripPrefix('c.', suffix);
```
(where `stripPrefix` is currently at https://github.com/Agoric/agoric-sdk/blob/db3403c45def2d5cec9eb3c16ec5d12439b1d93c/packages/SwingSet/src/kernel/state/kernelKeeper.js#L201 but could be moved)

Reply:
I don't really have an opinion. Iterating over a larger range and only considering a subset of it doesn't seem like a behavior worth keeping.